test/unit/__snapshots__/merchant-fulfillment.test.ts.snap

```diff
@@ -322,7 +322,7 @@ Array [
             "SellerInputDefinition": Object {
               "Constraints": Array [
                 Object {
-                  "ValidationRegEx": "^[a-zA-Z0-9-._&amp; ]{3,25}$",
+                  "ValidationRegEx": "^[a-zA-Z0-9-._& ]{3,25}$",
```

Comment:
This one get's parsed correctly too

`&amp;` = `&`

Reply:
Yeah, you are totally right! I overlooked that it was actually just escaping the literal `"` char. I thought it was the end of the string, and that `&quot;` was converted to `\` :D Good eye!

So does that mean that fast parser gets it wrong? Shall we open an issue in their repo about it?